back/internal/dbg/dbg.go

```diff
@@ -0,0 +1,59 @@
+package dbg
+
+import (
+	"io"
+	"log"
+	"sync/atomic"
+)
+
+// dbgLog is thread-safe.
+type dbgLog struct {
+	log *log.Logger
+	out io.Writer
+}
+
+var atomicV atomic.Value
```

Comment:
I suggest restructuring this design such that, in theory, the dbgLog type could be exported and have multiple instances existing in one application.

Reply:
I've renamed atomicV, moved it into the dbgLog struct, and changed what it stores.